drizzlepac/runastrodriz.py

```diff
@@ -709,7 +716,9 @@ def process(inFile, force=False, newpath=None, num_cores=None, inmemory=True,
             pass
 
     # Append final timestamp to trailer file...
-    _final_msg = '%s: Finished processing %s \n' % (_getTime(), inFilename)
+    end_time = _getTime()
+    _delta_time = time.time() - init_time
+    _final_msg = '%s: Finished processing %s in %.2f seconds \n' % (end_time, inFilename, _delta_time)
```

Comment:
Just curious why the initial time is in SSE, but the end time is in a human-readable time string?

Reply:
The (human readable) 'end_time' value is a string which can not be used for math operations, so the 'init_time' is used to compute the elapsed processing time instead.

Reply:
Ok.  Just a practical use of time formats.